packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -524,8 +524,6 @@ class CallbackAction<T extends Intent> extends Action<T> {
   CallbackAction({required this.onInvoke}) : assert(onInvoke != null);
 
   /// The callback to be called when invoked.
-  ///
-  /// Must not be null.
```

Comment:
Yes, but it's non-nullable, so I assume that gives enough implicit documentation that it can't be null.
I realize that someone could be using a weak nullability app, though.  I'll reinstate it.